### ISSUE TITLE
Devportal API Subscriptions page accessibility issues

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8374]
### Describe your problem(s)

1. buttons has no keyboard access
![image](https://user-images.githubusercontent.com/20179540/84800777-e9b85a00-b01b-11ea-98ec-5f8953da1a0d.png)

2. Form should have the required statement at the beginning

Required fields are marked with an asterisk ( * )

![image](https://user-images.githubusercontent.com/20179540/84802159-d27a6c00-b01d-11ea-89d8-c4d6b765cfe9.png)

3. first and input field reads the placeholder text as accessible output, not the input label

![image](https://user-images.githubusercontent.com/20179540/84851964-41d17980-b079-11ea-9459-42ea72dbb18c.png)

4. select fields has duplicate aria-labelledby ids

5. radio button does not has a keyboard focus

![image](https://user-images.githubusercontent.com/20179540/84855737-75fd6800-b082-11ea-9b83-5bde238cc61c.png)

6. access token field does not have a name and copy button is not focusable 

![image](https://user-images.githubusercontent.com/20179540/84855976-10f64200-b083-11ea-8882-83a390a1ab53.png)

7. Scopes text is not visible

![image](https://user-images.githubusercontent.com/20179540/84856105-5f0b4580-b083-11ea-9bb0-13c96bed41d7.png)

![image](https://user-images.githubusercontent.com/20179540/84856416-45b6c900-b084-11ea-9c4b-9d5887afb6ac.png)

8. delete confirmation modal should have `aria-alertdialog`

![image](https://user-images.githubusercontent.com/20179540/84857391-97f8e980-b086-11ea-8229-46fe071c8a84.png)

### How will you implement it

9. Add button is not visible

![image](https://user-images.githubusercontent.com/20179540/84865211-77845b80-b095-11ea-85cd-3f70bc01aaf9.png)

---
